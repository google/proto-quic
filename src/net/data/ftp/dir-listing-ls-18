-rw-r--r--    1 ftpadmin ftpadmin125435904 Apr  9  2008 .pureftpd-upload.47fcbb3c.849.191b.cd40d08a
-rw-r--r--    1 ftpadmin ftpadmin153198592 Nov 20  2008 .pureftpd-upload.4925a00d.849.668d.9ea5b3ed
-rw-r--r--    1 ftpadmin ftpadmin     8760 Nov 24  2008 .pureftpd-upload.492b0a03.849.12d.bf5d2bc6
