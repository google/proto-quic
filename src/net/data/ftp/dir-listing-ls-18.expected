-
.pureftpd-upload.47fcbb3c.849.191b.cd40d08a
-1
2008
4
9
0
0

-
.pureftpd-upload.4925a00d.849.668d.9ea5b3ed
-1
2008
11
20
0
0

-
.pureftpd-upload.492b0a03.849.12d.bf5d2bc6
8760
2008
11
24
0
0
